tracks/python/exercises/isbn-verifier/mentoring.md

```diff
@@ -0,0 +1,111 @@
+# Mentoring
+
+## Problem and Challenges
+
+The problem asks you to write an ISBN validation checker.
+
+* Any "-" in the input string should be ignored.
+* A valid ISBN must be 10 values long.
+* A valid ISBN must be entirely digits, other than the last character.
+* A valid ISBN may have `"X"` as the last character, which should be treated as a `10` value.
+* The ISBN is valid if `sum(value * position_counter) % 11 == 0` for `value in <input string>` and `position_counter in <(10..1)>`.
```

Comment:
This was intended as pseudo-code. Or, as a compact and succient manner to encapsulate the requirements here. I could write it out in English in a long form.

Reply:
```suggestion
* The ISBN is valid if the weighted sum of the digits is a multiple of 11 where the weight is the digit's value, ranging from 10 to 1.
```

Reply:
I think this is better than the former version.  Otherwise use variable names, and then explain what those variable names reference.  Programmer are used to interpreting variable names (especially if they are well named) and inferring what is meant.

But I like the written out explanation.  I can see the value of having the code presented to show the applied description.  

So, both?

Reply:
If you want working code which demonstrates what is needed ... isn't that what the next section contains?

Reply:
Then, yes, prefer the written here and the example in the next section.

Reply:
Done in this current iteration.